Pod/Classes/SEGAnalytics.h

```diff
@@ -88,6 +88,8 @@
 
  @param traits        A dictionary of traits you know about the user. Things like: email, name, plan, etc.
 
+ @param options       A dictionary of options, such as the `@"anonymousId"` key. If no anonymous ID is specified one will be generated for you.
+
```

Comment:
Yup! We might want to clarify in the future that not bundled integrations will respect this, but we'll see if this confuses users first.